src/dbnode/persist/fs/files_test.go

```diff
@@ -282,29 +282,38 @@ func TestTimeAndVolumeIndexFromFileSetFilename(t *testing.T) {
 }
 
 func TestFileExists(t *testing.T) {
-	dir := createTempDir(t)
-	defer os.RemoveAll(dir)
 
-	shard := uint32(10)
-	start := time.Now()
-	shardDir := ShardDataDirPath(dir, testNs1ID, shard)
-	err := os.MkdirAll(shardDir, defaultNewDirectoryMode)
+	var (
```

Comment:
mind making a new test which only tests the `checkpointFileSizeBytes` condition we're worried about and adding a little blurb about why it exists.